3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -41,6 +43,27 @@ enrich {
     secret-key: "iam"
   }
 
+  # config for NSQ sink and NSQ source
+NSQ {
+  #Topic name for good event source
+  good-source-topic: "{{sinkKinesisNsqSourceGoodTopicName}}"
+
+  #Channel name for good event source
+  good-source-channel: "{{sinkKinesisNsqSourceGoodChannelName}}"
+
+  #Topic name for bad event sink
+  bad-sink: "{{sinkKinesisNsqSinkTopicBadName}}"
```

Comment:
but this introduces two sources of truth and people won't know what is used so I suggest reusing everything and keeping the channel config here